tests/analyzers_manager/__init__.py

```diff
@@ -128,7 +128,7 @@ def test_start_analyzers(self, *args, **kwargs):
                     msg="all `analyzer_reports` status must be `SUCCESS`",
                 )
                 # check connectors status
-                if connectors_stats["all"] > 0 and connectors_stats["running"] == 0:
+                if connectors_stats["all"] >= 0 and connectors_stats["running"] == 0:
```

Comment:
should we include 0 ? that can raise a false positive in case no connectors were executed, no ?

Reply:
Hmm, this is also the case when no connectors are present

Reply:
but during testing, all connectors in the config should get executed regardless they are marked as disabled or not.

Reply:
I don't know why I was thinking that day, if someone removes all the connectors, the tests would start to fail.
Reverting this right away.